### PR TITLE
Scout minor release 4.22

### PR DESCRIPTION
**Pull request by northwestwitch**
New scout **minor** release 4.22

### Added
- Editable cases comments
- Editable variants comments
### Fixed
- Empty variant activity panel
- STRs variants popover
- Split new ClinVar multiple significance terms for a variant
- Edit the selected comment, not the latest
### Changed
- Updated RELEASE docs.
- Pinned variants card style on the case page
- Merged `scout export exons` and `scout view exons` commands

**Review:**
- [x] code approved by DN
- [x] tests executed by DN, CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
image

OOOOK...

**Comment by dnil** [comment]
Fingers crossed.. Hm, didn't we give all the dis a bypass at one point?

**Comment by northwestwitch** [comment]
> Fingers crossed.. Hm, didn't we give all the dis a bypass at one point?

There's always something you don't think about. Just embrace the concept

**Comment by northwestwitch** [comment]
Passed!

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,12 +6,18 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
...
+
+## [4.22]
+### Added
 - Editable cases comments
 - Editable variants comments
 ### Fixed
 - Empty variant activity panel
 - STRs variants popover
-- Split correctly ClinVar multiple significance terms for a variant 
+- Split correctly ClinVar multiple significance terms for a variant
```
Comment (dnil) [inline CHANGELOG.md:20]:
```suggestion
- Split new ClinVar multiple significance terms for a variant
```

**Review by dnil — APPROVED** [review]

**Comment by northwestwitch** [comment]
I'm installing it on stage to give it a final testing

**Comment by dnil** [comment]
Fingers crossed again.. Sorry, I forget quickly sometimes. The new version is on stage for testing!

**Comment by northwestwitch** [comment]
> Fingers crossed again.. Sorry, I forget quickly sometimes. The new version is on stage for testing!

Ah OK, you did it already, great!

**Comment by northwestwitch** [comment]
There is something fishy with comments. Let's wait for merging this

**Comment by northwestwitch** [comment]
Looks like if you have multiple comments and you want to edit an oldish one, it edits the latest instead.

**Comment by dnil** [comment]
Yep, same here for the case comments. Doesn't have to be old, more that the modal opens for the latest rather than the clicked one?

**Comment by northwestwitch** [comment]
> Yep, same here for the case comments. Doesn't have to be old, more that the modal opens for the latest rather than the clicked one?

I'm writing a fix

**Comment by northwestwitch** [comment]
Tested also the other features and fixes. I think that after the comment edit fix we're good to merge!

**Comment by northwestwitch** [comment]
Merging as soon as tests pass

**Pull request merged**